java/mdsobjects/src/test/java/MDSplus/MdsConnectionTest.java

```diff
@@ -55,7 +55,8 @@ public static void tearDownAfterClass() throws Exception {}
 	@Before
 	public void setUp() throws Exception 
 	{
-	    freePort= 8888;
+	    freePort= 8889; // there could be some issues here as it seems that if the port is
```

Comment:
would it not be easier to run mdsip explicitly. we have a mdsip.hosts prepared in ./testing/

Reply:
maybe use the same port range as we do in the cpp and python tests if not done already

Reply:
If you have a better way to call the mdsip server, that would be great !